packages/core/src/odata-v2/entity-serializer.spec.ts

```diff
@@ -111,43 +129,23 @@ describe('entity-serializer', () => {
     });
   });
 
-  it('should serialize an entity with complex type', () => {
-    const complexTypeProp = {
-      stringProperty: 'prop',
-      booleanProperty: false,
-      int16Property: 234
-    };
-    const stringProperty = 'code';
-
-    const entity = TestEntity.builder()
-      .complexTypeProperty(complexTypeProp)
-      .stringProperty(stringProperty)
-      .build();
-
-    expect(serializeEntity(entity, TestEntity)).toEqual({
-      ComplexTypeProperty: {
-        StringProperty: complexTypeProp.stringProperty,
-        BooleanProperty: complexTypeProp.booleanProperty,
-        Int16Property: complexTypeProp.int16Property
-      },
-      StringProperty: stringProperty
-    });
-  });
-
-  it('should serialize an entity with custom field', () => {
```

Comment:
I personally would keep these two tests.

Reply:
I moved them to common.